app/helpers/application_helper.rb

```diff
@@ -4,4 +4,16 @@ def categories
     Category.all
   end
 
+  def item_status_check(item)
```

Comment:
I've never used the helpers directory before. Thoughts on whether we should make an `items_helper.rb` or just leave it in `application_helper`?